examples/tutorial/tutorial_part_two.ipynb

```diff
@@ -69,7 +88,7 @@
    "source": [
     "# Train Logistic Regression Model\n",
     "\n",
-    "When tackling a new problem with machine learning, it's always good to start with simple models and only add complexity when needed. Start by training a logistic regression binary classifier using [Scikit-Learn](http://scikit-learn.org/). This model gets about 74% accuracy on the offline evaluation dataset."
+    "When tackling a new problem with machine learning, it's always good to start with simple models and only add complexity when needed. Start by training a logistic regression binary classifier using [Scikit-Learn](http://scikit-learn.org/). This model gets about 68% accuracy on the offline evaluation dataset if you use 10000 training datapoints about 74% if you use all training datapoints."
```

Comment:
Run on sentence. Change it to:
"This model gets about 68% accuracy on the offline evaluation dataset if you use 10000 training examples. It gets about 74% if you use all 50,000 examples."